docs/toedit-features.md

```diff
@@ -1,6 +1,6 @@
 ## JavaScript
 
-### ES Modules (ESM)
+### ES Modules (ESM) - Move to Getting Started
```

Comment:
@FredKSchott so since there isn't going to be a features page in this proposal, I went header by header and gave recommendations to where to move each section. Feedback appreciated!